Lib/cmd.py

```diff
@@ -273,7 +273,7 @@ def complete(self, text, state):
             endidx = readline.get_endidx() - stripped
             if begidx>0:
                 cmd, args, foo = self.parseline(line)
-                if cmd == '':
+                if cmd in ('', None):
```

Comment:
In this very specific case, I don't really like `in`. Maybe because `None` is part of it? I think `if cmd == '' or cmd is None` or `if cmd:` (flip the logic) are both slightly better.

Reply:
I don't like flipping the logic, it makes a future `git blame` harder to follow. I just went with `if not cmd:` instead

Reply:
That works for me.